selfdrive/common/params.cc

```diff
@@ -78,6 +79,12 @@ int write_db_value(const char* params_path, const char* key, const char* value,
     goto cleanup;
   }
 
+  // change permissions to 0644
+  result = fchmod(tmp_fd, 0644);
+  if (result < 0) {
+    goto cleanup;
+  }
```

Comment:
not needed, is it?

Reply:
chmod 0644 is needed for offroad app to read the params file

Reply:
ah, right. Haven't gone through tha APK/display part yet. Short term I'm fine making sure those params are logged, but yeah, would be nice to display them in developer settings.